r/tests/testthat/helper-skip.R

```diff
@@ -21,7 +21,10 @@ build_features <- c(
   uncompressed = TRUE
 )
 
+force_tests <- identical(tolower(Sys.getenv("ARROW_R_FORCE_TESTS")), "true")
```

Comment:
This is probably fine like it is, but if we wrapped this in a function, it would behave more like we expect — we wouldn't need to think about when this line is sourced

Reply:
I'll turn it into a function in case we move the helper stuff somewhere else at some point...